src/components/Jobs/jobs.util.js

```diff
@@ -271,10 +279,17 @@ export const enrichRunWithFunctionTag = (
       if (funcs) {
         const tagsList = uniq(map(funcs, 'metadata.tag'))
         set(jobRun, 'ui.functionTag', tagsList.join(', '))
+        set(jobRun, 'ui.runOnSpot', capitalize(funcs[0].spec.preemption_mode ?? ''))
+        set(jobRun, 'ui.nodeSelectorChips', parseKeyValues(funcs[0].spec.node_selector || {}))
+        set(
+          jobRun,
+          'ui.priority',
+          generateFunctionPriorityLabel(funcs[0].spec.priority_class_name ?? '')
+        )
       } else {
         set(jobRun, 'ui.functionTag', '')
       }
```

Comment:
if there is no function better to set empty values as it was done for `functionTag`. You can use `lodash.defaults` both in the `if` and `else` blocks